app/src/main/res/raw/SampleData.csv

```diff
@@ -0,0 +1,7 @@
+﻿Key,Name,Latitude,Longitude,Street Address,City,State,Zip,Type,Phone,Website
```

Comment:
I get an error on build that says res/raw files can't be uppercase but renaming to `sampledata.csv` worked